### ISSUE TITLE
The spatial attention module seems not implemented. 

### ISSUE THREAD

**Issue by KL4805** [MiaoHaoSunny/ST-DAAN#1]
The module for global spatial attention seems not implemented. Could you please check?

**Comment by MiaoHaoSunny** [comment]
Yes, I have checked the code roughly. The code placed in this repository was a initial verision that didn't have global spatial attention. The global spatial attention is a vey simple dot-product attention. Because that I have graduated from my school, it is a bit difficult for me to get the complete code. So if you are convenient, you can add the spatial attention directly. By the way, do you know Dr.Yu Yang(杨宇) from PolyU?

**Comment by 1997tzd** [comment]
Do you still have the complete code and data? I don't seem to find the PeopleDataLoader and Pretrain modules.
